client/src/pages/dashboard/teams/[teamId]/addPlayer.jsx

```diff
@@ -1,56 +1,139 @@
 import React from 'react'
```

Comment:
I'm not sure about the name of this file/route, we're adding multiple players right? Maybe something like `/add` would be more appropriate? I think using camelcase in a route name is a bit odd